### DISCUSSION TITLE
Inquiry 7

### DISCUSSION THREAD

**Discussion by yujispin990** [winstxnhdw/lc-hax#94] [Q&A] [answered]
What is the difference between NamInWeightMod & WeightMod? also how does the pumpkin kill people 😆.

**Comment by winstxnhdw** [comment, marked as answer]
If you look at people, you can see that their names and player ID replaces the weight label. That's the `NameInWeightMod`, and I am sure you already know what `WeightMod` does.

> also how does the pumpkin kill people

It just does 🤣
